lib/Certificate.php

```diff
@@ -140,6 +143,26 @@ public function isSelfSigned()
         return $this->info["subject"] === $this->info["issuer"];
     }
 
+    public function getKeySha256Base64(): string
+    {
+        return \base64_encode($this->getKeySha256Raw());
+    }
+
+    public function getKeySha256Hex(): string
+    {
+        return \bin2hex($this->getKeySha256Raw());
+    }
+
+    public function getKeySha256Raw(): string
```

Comment:
getSha256Pin? not sure we need the other methods btw

Reply:
Pinning is just one use case for the fingerprint.